Makefile

```diff
@@ -4,13 +4,22 @@ ROOT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 
 .PHONY: docs pylint flake8 virtualenv
 
+ifeq ($(OS),Windows_NT)
+  VENV_BIN_DIR := Scripts
+  VENV_PY_VERSION ?= python
+  VENV_CREATE := python -m venv
```

Comment:
I think we can just use venv on all platforms now. Does windows not call it python3 as well? I thought this would be consistent across platforms.

Reply:
No, The windows installer are not calling the executable python3.exe, only python.exe; It is a bit irritating, but I have to copy and rename the exe each time I do an installation.

Reply:
Alright, do you know how other package manager on windows handle it? i.e. does https://www.msys2.org/ also provide a python binary that is not python2?